dependency-check-maven-config.xml

```diff
@@ -192,4 +217,269 @@
+        <cve>CVE-2014-6060</cve>
+        <cve>CVE-2014-1939</cve>
+        <cve>CVE-2013-7372</cve>
+        <cve>CVE-2010-1807</cve>
+        <cve>CVE-2007-3150</cve>
+        <cve>CVE-2007-1085</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            This CVE is disputed by SUN, but the claim is that a denial of service attack can be incurred.
+            This dependency is used for workspace email notifications and is limited by the cron string configuration.
+            This cron string configuration mitigates the possibility of a denial of service attack.
+            javax.mail-1.5.5.jar
+            mail-1.4.5.jar
+        </notes>
+        <cve>CVE-2007-6059</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            This CVE applies to ganglia-web. OWASP is flagging this CVE on gmetric4j-1.0.7.jar (as a direct transitive and as a transitive to metrics-ganglia-3.2.2.jar).
+            Though gmetric4j-1.0.7.jar has the same group id as ganglia-web, it does not depend on ganglia-web.
+            gmetric4j-1.0.7.jar
+            metrics-ganglia-3.2.2.jar
+        </notes>
+        <cve>CVE-2007-6465</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            This CVE is at the OS level and does not apply to the running system.
+            jna-4.4.0.jar
+        </notes>
+        <cve>CVE-2012-0217</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            OWASP is looking at the version of Artemis rather than the version of ActiveMQ.
+            These CVEs do not apply to the version of ActiveMQ server that is used in the system.
+            artemis-core-client-2.1.0.jar
+        </notes>
+        <cve>CVE-2016-3088</cve>
+        <cve>CVE-2015-3208</cve>
+        <cve>CVE-2014-3576</cve>
+        <cve>CVE-2013-3060</cve>
+        <cve>CVE-2013-1880</cve>
+        <cve>CVE-2013-1879</cve>
+        <cve>CVE-2012-6551</cve>
+        <cve>CVE-2012-6092</cve>
+        <cve>CVE-2012-5784</cve>
+        <cve>CVE-2011-4905</cve>
+        <cve>CVE-2010-1244</cve>
+        <cve>CVE-2010-0684</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            This CVE applies to the Jolokia API provided by ActiveMQ which is secured by DDF security,
+            so the vulnerability is mitigated.
+            artemis-core-client-2.1.0.jar
+        </notes>
+        <cve>CVE-2015-5182</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            These CVEs apply to the Hawtio console which is not exposed in this system.
+            artemis-core-client-2.1.0.jar
+        </notes>
+        <cve>CVE-2015-5184</cve>
+        <cve>CVE-2015-5183</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            OWASP is looking at the version of Camel CXF rather than the version of Apache CXF.
+            These CVEs do not apply to the version of Apache CXF that is used in the system.
+            camel-cxf-2.19.0.jar
+        </notes>
+        <cve>CVE-2017-5656</cve>
```

Comment:
❓ Could we combine with those listed on L182-189?

Reply:
Done